TODO.md

```diff
@@ -0,0 +1,21 @@
+## Pomysły:
```

Comment:
to jest standardowa nazwa pliku na pomysły i tego typu rzeczy. `TODO.txt` jak to jest zwykły tekst, `TODO.md` jak chcesz bardziej zaawansowane formatowanie itp.